backend/models/__init__.py

```diff
@@ -3,4 +3,4 @@
 from models.user import User  # noqa: F401
 from models.prescription import Prescription  # noqa: F401
 from models.referral import Referral  # noqa: F401
-from models.medical_documentation import MedicalDocumentation  # noqa: F401
+from models.medical_documentation import MedicalDocumentation, MedicalDocumentationEntry  # noqa: F401
```

Comment:
Why did you add noqa F401 to every import?

Reply:
not every, but every in this file. It ensures that models are initialized before engine creates the database. Noqa F401 are just for removing flask warning